app/controllers/user_reservations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class UserReservationsController < ApplicationController
+  include JSONAPI::Fetching
+
+  def index
+    reservations = current_user.reservations
```

Comment:
It would be great to add policy from pundit here, it would allow you to have a single controller for both employees and users - normal user would get through that policy only their own reservations and employee would get all of them:
https://github.com/varvet/pundit#scopes

Reply:
I merged controllers and refactored policies, thanks.